controlplane/rosa/controllers/rosacontrolplane_controller.go

```diff
@@ -267,7 +268,7 @@ func (r *ROSAControlPlaneReconciler) reconcileNormal(ctx context.Context, rosaSc
 		billingAccount = rosaScope.ControlPlane.Spec.BillingAccount
 	}
 
-	spec := ocm.Spec{
+	ocmSpec := ocm.Spec{
```

Comment:
clusterSpec is the better name

Reply:
clusterSpec might be confused with CAPI cluster spec. the name should be specific to our implementation.

Reply:
better then ocmClusterSpec. ocmSpec is confusing as well ocm is not an object